ang/civicase/case/list/directives/case-list-table.directive.js

```diff
@@ -167,7 +167,7 @@
         } else {
           $scope.viewingCaseDetails = _.findWhere($scope.cases, { id: id });
           $scope.viewingCase = id;
-          $scope.viewingCaseTab = 'summary';
+          $scope.viewingCaseTab = 'Summary';
```

Comment:
Hmm, how did it work before 😕 ?

Reply:
It didn't..... since https://github.com/compucorp/uk.co.compucorp.civicase/pull/281 has not been QA tested this issue was not found.